moztrap/view/manage/cases/forms.py

```diff
@@ -109,7 +109,14 @@ def is_valid(self):
     def save_attachments(self, caseversion):
         # @@@ convert into a modelmultiplechoicefield widget?
         delete_ids = set(self.data.getlist("remove-attachment"))
-        caseversion.attachments.filter(id__in=delete_ids).delete()
+
+        # TODO!
```

Comment:
should we enter a bug for this?  not sure.  maybe not necessary if the functionality is still working ok.